.github/workflows/build.yml

```diff
@@ -37,8 +38,9 @@ jobs:
           GH_BUILD_USERNAME: ${{ secrets.GH_BUILD_USERNAME }}
           GH_BUILD_PASSWORD: ${{ secrets.GH_BUILD_PASSWORD }}
       run: |
-        make TARGET=ECW5410
         TAG=$(date +%Y-%m-%d)-$(git rev-parse --short HEAD)
+        echo "ecw5410-$TAG" > feeds/wlan-ap/opensync/src/vendor/ecw5410/.pkgname
+        make TARGET=ECW5410
```

Comment:
Should we not just make the opensync Makefile add this ? running make twice seems sub optimal

Reply:
sorry, iI did not have enough coffee yet, i missed the prior removal line ;)

Reply:
> sorry, iI did not have enough coffee yet, i missed the prior removal line ;)

No problem John :)